### DISCUSSION TITLE
Packet rerouting, v/s forwarding, v/s switching, v/s redirecting

### DISCUSSION THREAD

**Discussion by arayu05** [cilium/ebpf#528] [Q&A] [answered]
Hi Team,
I have a very basic question.
Can someone please explain the difference b/w below terms:

Packet rerouting,
Packet forwarding, 
Packet switching, 
Packet redirecting

**Comment by ti-mo** [comment, marked as answer]
Hi @arayu05, I'm not sure how this question is relevant to this repository. Closing.
